readme-vars.yml

```diff
@@ -7,6 +7,7 @@ project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/ma
 project_blurb: |
   [{{ project_name|capitalize }}]({{ project_url }}) is an open source personal relationship management system, that lets you document your life.
 project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+project_categories: "Content Management"
```

Comment:
Not really content management, more CRM (or FRM I guess in this case), not sure

Reply:
Not sure if we should create another category for it